kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/util/JsonUtils.java

```diff
@@ -33,7 +33,7 @@ public static String toJson(Object object) {
     }
   }
 
-  public static <T> T toObject(String json, Class<T> clazz) {
+  public static <T> T fromJson(String json, Class<T> clazz) {
```

Comment:
A simple rename, and this way can not cover generic classes, e.g. `List<String>` because of the restrictions on Java generics system. But it's OK for now.